tests/E2E Tests/TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -92,6 +99,24 @@ public async Task AcquireToken_WithMicrosoftIdentityApplicationOptions_ClientCre
             await CreateGraphClientAndAssert(tokenAcquirerFactory, services);
         }
 
+        [IgnoreOnAzureDevopsFact]
+        //[Fact]
+        public async Task AcquireToken_WithMicrosoftIdentityApplicationOptions_ClientCredentialsCiamAsync()
+        {
+            TokenAcquirerFactory tokenAcquirerFactory = TokenAcquirerFactory.GetDefaultInstance();
+            IServiceCollection services = tokenAcquirerFactory.Services;
+
+            services.Configure<MicrosoftIdentityApplicationOptions>(s_optionName, option =>
+            {
+                option.Instance = "https://MSIDLABCIAM6.ciamlogin.com";
```

Comment:
With CIAM we would not be supporting the `Instance` and `TenantId` configuration. We expect ppl to configure `Authority` instead.